api-reference/headless-portal.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: 'Headless Portal'
+description: 'Give your customers visibility over their support requests.'
+---
+
+Customer support portals empower your customers to view, create and reply to support requests as well as share visibility of issues with their team.
+
+Pre-built support portals can be hard to customize or theme and have to be separate from your product. Instead Plain provides a set of powerful APIs as well as an [example repository](https://github.com/team-plain/plain-headless-portal) which allows you to easily create a fully custom support portal.
+
+In order to use the Plain API you will need an API key, see [Getting Started](https://github.com/team-plain/plain-headless-portal?tab=readme-ov-file#getting-started) for full instructions. This API key should be kept secret since it will allow access to data from your whole Plain workspace. Calls to Plain's API **should be made from a server** rather than the client.
+
+## Fetching threads
+
+To display a list of support requests you will need to fetch the appropriate threads from your workspace. The recommended way to do this is to filter by [tenant](/api-reference/graphql/tenants).
+
+```Javascript
+  const threads = await plainClient.getThreads({
+    filters: {
+      tenantIdentifiers: [{ externalId: tenantExternalId }],
+      statuses: [ThreadStatus.Todo, ThreadStatus.Snoozed],
+    },
+  });
+```
+
+Tenants allow you to structure your customers in Plain in the same way as they are structured in your product. For example if within your product customers are organised in a ‘team’ then you would create one tenant per team in your product.
```

Comment:
I think this should be in italic to indicate that is isn't the same topic as the rest of the article